test/services/teams/helper/helper.format.js

```diff
@@ -0,0 +1,12 @@
+exports.h1 = (text) => {
```

Comment:
IMHO this is unnecessary, as Mocha/chai already formats the test reports well enough if you use nested `describe` blocks

Reply:
also true